app/src/debug/java/net/squanchy/support/debug/DebugActivity.kt

```diff
@@ -134,4 +143,38 @@ class DebugActivity : AppCompatActivity() {
         OnboardingResetter(this).resetOnboarding()
         Snackbar.make(findViewById<View>(R.id.debug_root), "It's daaaawnnnn", Snackbar.LENGTH_SHORT).show()
     }
+
+    private fun freezeTime() {
+        val now = DebugCurrentTime(applicationContext).currentDateTime()
+
+        pickDate(now)
+    }
+
+    private fun pickDate(now: ZonedDateTime) {
+        val datePickerDialog = DatePickerDialog(this, DatePickerDialog.OnDateSetListener { _, year, month, dayOfMonth ->
+            pickTime(now, LocalDate.of(year, month.to1Based(), dayOfMonth))
+        }, now.year, now.monthValue.to0Based(), now.dayOfMonth)
+
+        datePickerDialog.show()
+    }
+
+    private fun Int.to1Based() = this + 1
+
+    private fun Int.to0Based() = this - 1
+
+    private fun pickTime(now: ZonedDateTime, frozenDate: LocalDate) {
+        val timePickerDialog = TimePickerDialog(this, TimePickerDialog.OnTimeSetListener { _, hourOfDay, minute ->
```

Comment:
Whatever happened to one argument per line? :D

Reply:
abf04c9